### ISSUE TITLE
EfficientDet_d0 stops in colab while training

### ISSUE THREAD

**Issue by Vikram12301** [tensorflow/models#10997]
On finetuning the model in google colab, the model stops without any reason

**INFO:tensorflow:Step 800 per-step time 0.182s
I0519 18:26:33.234112 140311211771712 model_lib_v2.py:705] Step 800 per-step time 0.182s
INFO:tensorflow:{'Loss/classification_loss': 0.6315968,
 'Loss/localization_loss': 0.015179463,
 'Loss/regularization_loss': 0.030118918,
 'Loss/total_loss': 0.6768952,
 'learning_rate': 0.02628}
I0519 18:26:33.236839 140311211771712 model_lib_v2.py:708] {'Loss/classification_loss': 0.6315968,
 'Loss/localization_loss': 0.015179463,
 'Loss/regularization_loss': 0.030118918,
 'Loss/total_loss': 0.6768952,
 'learning_rate': 0.02628}
^C**
